src/main/java/frc/robot/Robot.java

```diff
@@ -35,6 +35,7 @@
     public static Limelight limelight;
     public static RobotConstants robotConstants;
     public static FieldConstants fieldConstants;
+    private OI oi;
```

Comment:
move oi up to where we declare private fields

Reply:
`try`